cmd/edv-rest/startcmd/start.go

```diff
@@ -81,7 +81,7 @@ const (
 	databaseTimeoutEnvKey        = "EDV_DATABASE_TIMEOUT"
 	databaseTimeoutFlagShorthand = "o"
 	databaseTimeoutFlagUsage     = "Total time in seconds to wait until the database is available before giving up." +
-		" Default: " + string(rune(databaseTimeoutDefault)) + " seconds." +
```

Comment:
This `string(rune(value))` doesn't seem to work - the value doesn't print correctly. I replaced it with the actual value in the string. Is there a better way to fix this? I can't call strconv in a string, and the only other thing I could think of was to define the number as a string, but then this means doing converting the number from a string to an integer in the code, which also seems a bit ugly.